cpp/util/etcd.cc

```diff
@@ -180,6 +191,42 @@ void GetRequestDone(const string& keyname, EtcdClient::GetResponse* resp,
 }
 
 
+void CopyStat(const string& key, const JsonObject& from,
+              map<string, int64_t>* to) {
+  CHECK_NOTNULL(to);
+  const JsonInt stat(from, key.c_str());
+  if (!stat.Ok()) {
+    LOG(WARNING) << "Failed to find stat " << key;
+  }
+  (*to)[key] = stat.Value();
```

Comment:
Should you set it if it doesn't exist? I'm not sure what this does with a null...